google-client-loader.html

```diff
@@ -195,12 +144,27 @@
         }
       },
 
+      /**
+       * Wrapper for `gapi.auth`.
+       * @property auth
+       * @type gapi.auth
+       */
+      get auth() {
```

Comment:
getters were more of an issue in 0.5 when Object.observer() wasn't present. Think it's fine to leave, but I'll leave it up to you.